finder-packages.php

```diff
@@ -6,6 +6,7 @@
         'source/woocommerce/packages/action-scheduler/classes',
```

Comment:
could you move the new line here
no it looks line the comment is also for that line

Reply:
Yes of course